src/Controller/QueueTrigger.php

```diff
@@ -24,14 +25,17 @@ public function processToEndTime() {
       throw new NotFoundHttpException();
     }
     else {
-      // The endpoint has been called successfully.
-      // @Todo Call the queue runner service.
-      // Log it.
-      $vars = ['@time' =>  date('m-d-Y H:i:s', $time)];
+      // Process our bulletins.
+      $trigger = new BulletinBase();
```

Comment:
```suggestion
      $QueueWorker = new BulletinBase();
```
Maybe?

Reply:
Changing to $queueWorker - starting with an uppercase makes cs complain, @swirtSJW